property/index.js

```diff
@@ -93,6 +96,7 @@ module.exports = yeoman.generators.Base.extend({
         name: 'itemType',
         message: 'The type of array items:',
         type: 'list',
+        default: this.propDefinition && this.propDefinition.itemType,
```

Comment:
see comment below (setting this.propertyDefinition.required broke the behaviour)